consensus/spos/bls/subroundBlock.go

```diff
@@ -548,17 +561,19 @@ func (sr *subroundBlock) saveProofForPreviousHeaderIfNeeded() {
 		return
 	}
 
-	proof := sr.Blockchain().GetCurrentHeaderProof()
-	if !isProofEmpty(proof) {
+	proof, err := sr.worker.GetEquivalentProof(sr.GetData())
```

Comment:
I think this can be removed completely L564-576.. reason: it should be saved into the newly added equivalentProofsTracker and this will be done from the interceptor

will be done in a further PR, maybe just save it into equivalentProofsTracker for this one

Reply:
done